src/core/lib/surface/passive_listener_internal.h

```diff
@@ -11,18 +11,19 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-#ifndef GRPC_SRC_CPP_SERVER_PASSIVE_LISTENER_INTERNAL_H
-#define GRPC_SRC_CPP_SERVER_PASSIVE_LISTENER_INTERNAL_H
+#ifndef GRPC_SRC_CORE_LIB_SURFACE_PASSIVE_LISTENER_INTERNAL_H
+#define GRPC_SRC_CORE_LIB_SURFACE_PASSIVE_LISTENER_INTERNAL_H
 
 #include <grpc/support/port_platform.h>
 
+#include <grpc/passive_listener.h>
 #include <grpc/passive_listener_injection.h>
-#include <grpcpp/passive_listener.h>
 
 #include "src/core/lib/surface/server.h"
 
-namespace grpc {
+namespace grpc_core {
 namespace experimental {
+
 // A PIMPL wrapper class that owns the only ref to the passive listener
 // implementation. This is returned to the application.
 class PassiveListenerOwner final : public PassiveListener {
```

Comment:
This class isn't needed in C-core, only at the C++ layer.  Suggest moving this into server_builder.cc.